blog-posts/emails/article.md

```diff
@@ -0,0 +1,329 @@
+}
+```
+
+Great! You are now sure that your mails will be sent 🙃. But will your users receive them in their inbox? Probably not all of them if you don't customize your configuration a bit.
+
+## 2. Make sure you don't end up in your users spams
+
+When sending a user receives an email, there are many factors at stake, that will determine if the email ends up in the mailbox or in the spams.
+
+During my experience developing an email-sending service, I discovered [mail-tester][mail-tester]: a great tool to evaluate the probability of emails ending up in a inbox. Just send a test email to the address they give you, and it will analyze the likelihood of your message being well received.
+
+![Mail tester bad](./assets/mail-tester-bad.png 'Mail tester bad')
+
+This first mark is terrible, but you can fix it using the tips given my the website!
+
+Many factors determine if your email will be considered suspicious, let me cover the most important ones.
+
+### Email content
+
+The easiest to fix (it may already be OK on your app), but also the most important. Mail clients will often automatically classify your email as spam if you don't comply.
+
+- Your email should absolutely **have a subject and a body**.
+- Your email **shouldn't be too short**, especially if it contains images.
+- If your email contains links, prefer using **full links instead of minified ones**.
+- Also, check that every link included in your email is healthy and **redirects to a working website**.
+- Finally, a `List-Unsubscribe` header would improve your reputation, especially if you plan to send promotional emails. You will see in part 3 of this post how to implement it.
+
+### Sender domain
+
+The second thing checked by mail clients is the web domain where the email originates from. If you use a single existing address (like a gmail address) as identity to send your messages, they will originate from the AWS domain name, which is sometimes considered as suspicious.
+
+My advice is to **use your own domain name to send emails**. If you don't already own one, you can easily buy it for a fair price on **AWS Route 53**. Working with your own domain name will allow you to fine tune its DNS to become a trusted sender.
+
+Let's then create a new identity in AWS SES: you want to create a domain identity, with the domain name corresponding to what you just reserved on AWS Route 53. Let's also asign your configuration set, that is linked to the cloudwatch alarms and suppression list previously set up.
+
+![SES Domain Identity](./assets/ses-domain-identity.png 'SES Domain Identity')
+
+Then, a key feature of domain based identities is the ability to specify a custom MAIL FROM domain, that will increase the mailing clients trust. Choose a subdomain of your domain like `notifications.<your_domain>`. You can leave default behaviour on MX failure and if your domain is hosted on Route 53, click the last checkbox to let AWS automatically do the job of updating the domain's DNS.
```

Comment:
Using a subdomains adds clarity and trust for end readers, but it also allows you to separate sender reputations by subdomains, and thus avoid affecting your "whole" domain.

--> you do not impact your transactional emails if, for example, you screw up your promotional emails